shutdown.c

```diff
@@ -26,6 +26,12 @@ class cDBusShutdownHelper
     g_variant_builder_unref(builder);
   };
 
+  static void UserActive(cDBusObject *Object, GVariant *Parameters, GDBusMethodInvocation *Invocation)
```

Comment:
I think the function should be named `IsUserActive`
```suggestion
  static void IsUserActive(cDBusObject *Object, GVariant *Parameters, GDBusMethodInvocation *Invocation)
```